catalog/app/containers/Bucket/requests.js

```diff
@@ -1051,6 +1053,59 @@ export const getRevisionData = async ({
   }
 }
 
+export async function loadManifest({
+  s3,
+  bucket,
+  name,
+  revision = 'latest',
+  maxSize = MANIFEST_MAX_SIZE,
+}) {
+  try {
+    const { hash } = await loadRevisionHash({ s3, bucket, name, id: revision })
+    const manifestKey = `${MANIFESTS_PREFIX}${hash}`
+
+    if (maxSize) {
+      const h = await s3.headObject({ Bucket: bucket, Key: manifestKey }).promise()
+      if (h.ContentLength > maxSize) {
+        throw new errors.ManifestTooLarge({
+          bucket,
+          key: manifestKey,
+          maxSize,
+          actualSize: h.ContentLength,
+        })
+      }
+    }
+
+    const m = await s3.getObject({ Bucket: bucket, Key: manifestKey }).promise()
+    const [header, ...rawEntries] = pipeThru(m.Body.toString('utf-8'))(
+      R.split('\n'),
+      R.map(tryParse),
+      R.filter(Boolean),
+    )
+    const meta = header.user_meta
+    const entries = pipeThru(rawEntries)(
+      R.map((e) => [
+        e.logical_key,
+        {
+          hash: e.hash.value,
+          physicalKey: e.physical_keys[0],
+          size: e.size,
+          meta: e.meta,
+        },
+      ]),
+      R.fromPairs,
+    )
+    return { meta, entries }
+  } catch (e) {
+    if (e instanceof errors.ManifestTooLarge) throw e
+    // eslint-disable-next-line no-console
+    console.log('loadManifest error:')
+    // eslint-disable-next-line no-console
+    console.error(e)
+    throw e
```

Comment:
Seems like you can do `console.log` for `ManifestTooLarge` too, and remove condition

Reply:
`ManifestTooLarge` is expected and we show the appropriate message in the UI, so i see no point in logging it to the console

Reply:
Actually, I think `console.log` shouldn't be here at all. It's a cosmetic thing and should be on the last step of handling error